bioimageio/spec/model/v0_5.py

```diff
@@ -276,51 +311,48 @@ class WithHalo(Node):
 class BatchAxis(AxisBase):
     type: Literal["batch"] = "batch"
     id: Annotated[AxisId, Predicate(lambda x: x == AxisId("batch"))] = AxisId("batch")
-    size: Optional[Literal[1]] = None
+    batch_size: Optional[Literal[1]] = None
```

Comment:
> how about we just use `ParameterizedSize(min=1, step=1)` as a default value and leave the field as `size`?

But this would type this field as `Optional[ParameterizedSize]`, which would allow it to be something like `ParameterizedSize(min=3, step=10)`, and that seems to go against the original intention of having it be either exactly 1 or any number picked by the runtime

Reply:
Ha, true! 
How about 
```python
class ParameterizedBatchSize(ParametrerizedSize):
   min: Literal[1]
   step: Literal[1]
```
and
```python
class BatchAxis....
    size: Union[Literal[1], ParameterizedBatchSize] = ...
```
?